build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:2.3.0'
```

Comment:
It seems that 2.3.0 don't break something.
I've also nothing read about "2.3.0 the horror release" (or similar 😺)
So I approve it and merge it 🎉